src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/BackupConfiguration.kt

```diff
@@ -0,0 +1,49 @@
+package com.atlassian.performance.tools.jiraactions.api.page
+
+import com.atlassian.performance.tools.jiraactions.Patience
+import org.apache.logging.log4j.LogManager
+import org.openqa.selenium.By
+import org.openqa.selenium.WebDriver
+
+internal class BackupConfiguration(
+    private val driver: WebDriver,
+    private val access: AdminAccess
+) {
+    private val logger = LogManager.getLogger(this::class.java)
+    private val backupLocator = By.xpath(
+        "//*[@id='tbl_services']" +
+            "//*[contains(text(), 'com.atlassian.jira.service.services.export.ExportService')]" +
+            "/ancestor::tr" +
+            "//*[contains(@class, 'operations-list')]" +
+            "//*[contains(text(), 'Delete')]"
+    )
+
+    fun disable(): BackupConfiguration {
+        val prompted = access.isPrompted()
+        if (prompted) {
+            access.gain()
+        }
+        if (isBackupServicePresent()) {
+            ensureSwitchIsOff()
+        } else {
+            logger.info("This Jira does not support Backups during the tests configuration")
+        }
+        if (prompted) {
+            access.drop()
+        }
+        return this
+    }
+
+    private fun isBackupServicePresent(): Boolean {
+        return Patience().test { driver.isElementPresent(backupLocator) }
+    }
+
+    private fun ensureSwitchIsOff() {
+        driver.findElements(backupLocator).forEach { it.click() }
```

Comment:
The integration test should prepare multiple backup services, because it seems we're testing with only one.